### ISSUE TITLE
Version bump/PyPI update

### ISSUE THREAD

**Issue by threecgreen** [alvinwan/TexSoup#45]
The latest version of TexSoup on PyPI is from April and there have been several commits since then, including @SimonMaenaut's new features. All tests and checks are passing, why not bump the version to 1.4 and update PyPI.

**Comment by tomneep** [comment]
Firstly, thanks for creating this @alvinwan. I just discovered this and I think it is going to be very useful for a project I am working on. 

I'd like to second @threecgreen's request to bump the version and update PyPI. It seems that a few of the recently open issues have been fixed in the master. I tested #51, #53, #54 and all the examples given seemed to work fine with the master branch. @SimonMaenaut has already responded to say that #49 works on the master branch. I suspect #50 might too, but a more specific example would be helpful.
